[CI] IntervalThrottlerTests testThrottling failing

**Build Scans:**
- [elasticsearch-periodic #5817 / openjdk21_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/qstuzgwccgmne)
- [elasticsearch-pull-request #49473 / part-1](https://gradle-enterprise.elastic.co/s/rqcuxyz2dqzja)

**Reproduction Line:**
```
./gradlew ":server:test" --tests "org.elasticsearch.index.mapper.IntervalThrottlerTests.testThrottling" -Dtests.seed=F521CB3BFD716F16 -Dtests.locale=en-LR -Dtests.timezone=America/Chihuahua -Druntime.java=21
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.index.mapper.IntervalThrottlerTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testThrottling),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: null
```

**Issue Reasons:**
- [main] 2 failures in test testThrottling (0.3% fail rate in 712 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/server/src/test/java/org/elasticsearch/index/mapper/IntervalThrottlerTests.java
+++ b/server/src/test/java/org/elasticsearch/index/mapper/IntervalThrottlerTests.java
@@ -14,12 +14,12 @@ import org.elasticsearch.test.ESTestCase;
 public class IntervalThrottlerTests extends ESTestCase {
 
     public void testThrottling() throws Exception {
-        var throttler = new IntervalThrottler.Acceptor(10);
+        var throttler = new IntervalThrottler.Acceptor(100);
         assertTrue(throttler.accept());
         assertFalse(throttler.accept());
         assertFalse(throttler.accept());
 
-        Thread.sleep(20);
+        Thread.sleep(110);
         assertTrue(throttler.accept());
         assertFalse(throttler.accept());
         assertFalse(throttler.accept());

